src/TreeView/TreeView.tsx

```diff
@@ -265,7 +269,8 @@ const Item = React.forwardRef<HTMLElement, TreeViewItemProps>(
                   bg: 'accent.fg',
                   borderRadius: 2
                 }
-              }
+              },
+              ...sx
```

Comment:
Should we do a deep merge of the provided `sx` prop?

Reply:
Definitely! Good call